### ISSUE TITLE
Quick & Easy solution for this

### ISSUE THREAD

**Issue by Lastorder-DC** [WinampDesktop/winamp#1113]
1. Change License to GPLv3 now(GPLv2, LGPLv2, etc... all lower version can be sublicenced to higher version as I know)
2. Close Issues & PR(since there will be spams, etc. If devs are not managing)
3. Let "Community" do their job. naming branch to "community" is not proper way.

**Comment by Greeniac1** [comment]
I don't think that they should close the issues, people are trolling/spamming because of how "bad" the repo is, once they fix it, the spam will be reduced.

**Comment by Lastorder-DC** [comment]
I think spam will not go away for a while since there are already 700+ issues not related about Winamp(at least in constructive way) we can discuss at one of forks that will moderate spams more than this.

at least they should apply `Temporary interaction limits` (`Limit to existing users`) to prevent more spam coming.
